content/nextflow-on-aws-batch/30_batch/50_iam_batchrole.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "AWSBatch Service Role"
+chapter: false
+weight: 50
+---
+
+## Attach S3Admin to Batch Role
+
+In order to upload and download from S3 within a job, the AWSBatchServiceRole needs updating.
+Click [this deep link](https://console.aws.amazon.com/iam/home#/roles/AWSBatchServiceRole) to reach the *AWSBatchServiceRole* IAM Roles.
```

Comment:
add "make sure you're in the right region in the console, where you're running the workshop"